lib/nerve/service_watcher.rb

```diff
@@ -111,10 +117,14 @@ def run()
       until watcher_should_exit? || repeated_report_failures >= @max_repeated_report_failures
         report_succeeded = check_and_report
 
-        if report_succeeded
+        case report_succeeded
+        when true
           repeated_report_failures = 0
-        else
+        when false
           repeated_report_failures += 1
+        when nil
+          # this case exists for when the request is throttled
```

Comment:
can we add some logs here? when we first rollout, this is very important to have logs to monitor behavior.

Reply:
The `check_and_report` emits a metric for when a request is throttled and also logs that. I can also log that we hit this case; are there other logs that would be helpful?

Reply:
can we change `debug` to `info`? In production, only info level is enabled. this line is hit only on-demand, so log volume shouldn't be concern.